Start talking about variables and data types

Glitchy-Fire opened the pull request:
This isn't done yet! I started this too late (It's 1:15 right now), but I'll try to finish it later. Feel free to change the explanations or edit what's necessary. what's clear to me might not be clear to other people. Also, if this is a course for people of all skill levels, I'm debating on whether or not to put a section dedicated to explaining what exactly programming is. Not sure if this will fit or not though, it's up to you

Glitchy-Fire committed:
Start talking about variables and data types

Editor's Note: This isn't done yet! I started this too late (It's 1:15 right now), but I'll try to finish it later. Feel free to change the explanations or edit what's necessary. what's clear to me might not be clear to other people. Also, if this is a course for people of all skill levels, I'm debating on whether or not to put a section dedicated to explaining what exactly programming is. Not sure if this will fit or not though, it's up to you

TD-QM commented on CS_Ed_Week.md:
```diff
@@ -11,6 +11,34 @@ A workshop for getting young people interested in computer science
+- Integers are just what they sound like. 
+- This data type stores numbers, specifically whole numbers as that's what an integer is.
+- They can be edited using standard math operators, such as addition, subtraction, multiplcation, and division.
+  - It's important to note that when doing division on numbers that don't go into each other, the data type switches to a float which will be discussed later
```
double, not float

Glitchy-Fire committed:
Update CS_Ed_Week.md

cuppajoe123 commented on CS_Ed_Week.md:
```diff
@@ -11,6 +11,40 @@ A workshop for getting young people interested in computer science
+### Boolean
+- Take a simple light switch.
+- Standard light switches have two states: on or off.
+- We store either "on or off" using a boolean varibale, but instead of using "on or off," we store "true or false".
```
one suggestion I have is to add that 1 evaluates to true, and 0 evaluates to false

Glitchy-Fire commented:
Updated the boolean description to have a tie to binary. Also deleted the section about floats and doubles since those are too complex. Just the bare bones for now.

Glitchy-Fire committed:
I forgot this step, fml
